ovs-p4rt/sidecar/ovsp4rt.cc

```diff
@@ -375,8 +381,8 @@ void PrepareFdbTableEntryforV4VxlanTunnel(
           param->set_param_id(GetParamId(
               p4info, L2_FWD_TX_TABLE_ACTION_POP_VLAN_SET_VXLAN_UNDERLAY_V6,
               ACTION_PARAM_TUNNEL_ID));
-          // TODO(derek): 8-bit vni (es2k, ipv6)
-          param->set_value(EncodeByteValue(1, learn_info.tnl_info.vni));
+          // tunnel_id is bit<20>
```

Comment:
Is this comment correct? IPv6 uses 20-bit VNI instead of 24-bit? Please double-check

Reply:
I'm positive.

```json
actions {
  preamble {
    id: 23849990
    name: "linux_networking_control.pop_vlan_set_vxlan_underlay_v6"
    alias: "pop_vlan_set_vxlan_underlay_v6"
  }
  params {
    id: 1
    name: "tunnel_id"
    bitwidth: 20
  }
}
```

Reply:
I mentioned this in the PR comment:

> Implemented a new `EncodeTunnelId()` function to use in place of `EncodeVniValue()` when the destination is a tunnel_id field. It encodes the low-order 20 bits of a uint32_t parameter. _Note that these VNI values are 20 bits, not 24._

Reply:
Stuff like this is why I'm increasingly concerned about the lack of a test suite.